doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -13,39 +13,179 @@ The library supports the following technologies:
+         .. code-block:: console
 
-Before you can use this library, you must create a *Thing* for your client in AWS IoT.
-The *Thing* must be connected to a security policy.
-For testing, you can use a permissive policy, but make sure to update the policy to be more restrictive before you go into production.
-See `AWS IoT Developer Guide: Basic Policy Variables`_ and `AWS IoT Developer Guide: Security Best Practices`_ for more information about policies.
+            aws iot create-keys-and-certificate --set-as-active --certificate-pem-outfile device_cert.pem --public-key-outfile pub_key.pem --private-key-outfile priv_key.pem
 
-To create a Thing for your device:
+      #. Take note of the certificate ARN from the previous command, as it is required later.
+      #. Obtain a list of installed keys with
 
-1. Log in to the `AWS IoT console`_.
-#. Go to :guilabel:`Security` > :guilabel:`Policies` and select :guilabel:`Create policy`.
-#. Enter a name and define your policy.
-   For testing purposes, you can use the following policy (click on :guilabel:`JSON`, copy, and paste it):
+         .. code-block:: console
+
+            nrfcredstore <serial port> list
+
+         where ``<serial port>`` corresponds to the serial port of your device.
+
+      #. Choose a security tag that is not yet in use and thus does not show up in the list of installed keys.
+      #. Provision the client certificate using
+
+         .. code-block:: console
+
+            nrfcredstore <serial port> write <sec tag> CLIENT_CERT device_cert.pem
+
+	      where ``<serial port>`` is the serial port of your device and ``<sec tag>`` is the previously chosen unused security tag.
+
+      #. Provision the client key using
+
+         .. code-block:: console
+
+            nrfcredstore <serial port> write <sec tag> CLIENT_KEY priv_key.pem
+
+	      where ``<serial port>`` is the serial port of your device and ``<sec tag>`` is the previously chosen unused security tag.
+
+      #. Download the `Amazon Root CA 1`_ PEM file.
+      #. Provision the certificate using
+
+         .. code-block:: console
+
+            nrfcredstore <serial port> write <sec tag> ROOT_CA_CERT AmazonRootCA1.pem
+
+         where ``<serial port>`` is the serial port of your device and ``<sec tag>`` is the previously chosen unused security tag.
+
+   .. tab:: nRF70: Keys generated by AWS
+
+      To obtain a key pair and certificate generated by AWS, perform the following steps:
+
+      1. Generate the key pair and certificate with:
+
+         .. code-block:: console
+
+            aws iot create-keys-and-certificate --set-as-active --certificate-pem-outfile device_cert.pem --public-key-outfile pub_key.pem --private-key-outfile priv_key.pem
+
+      #. Take note of the certificate ARN from the previous command, as it is required later.
+      #. Download the `Amazon Root CA 1`_ PEM file.
+      #. Provision the certificates and private key at runtime, as shown in the :ref:`aws_iot` sample.
```

Comment:
You can move the documentation here regarding `CONFIG_AWS_IOT_PROVISION_CERTIFICATES`. I think its fine that we document how provisioning occurs in both the sample and application.

Reply:
The last step now says

>       #. Provision the certificates and private key at runtime to the Mbed TLS stack.
>           This is achieved by placing the PEM files into a :file:`certs/` subdirectory and ensuring the :kconfig:option:`CONFIG_AWS_IOT_PROVISION_CERTIFICATES` Kconfig option is enabled.
>           For more information, refer to the :ref:`aws_iot` sample as well as the :kconfig:option:`CONFIG_AWS_IOT_CERTIFICATES_FILE` Kconfig option.

Do you think that is sufficient? That should also cover the previous comment?

Reply:
that works!